docker/stacks/local/sqlite/docker-compose.yml

```diff
@@ -0,0 +1,94 @@
+version: "3.3"
+
+services:
+
+  mysql:
+    image: mariadb:10.6
+    restart: unless-stopped
+    environment:
+      - MYSQL_ROOT_PASSWORD=qHZDZ26fPqqCfKZtWkQ9
+      - MYSQL_DATABASE=pda
+      - MYSQL_USER=pda
+      - MYSQL_PASSWORD=qHZDZ26fPqqCfKZtWkQ9
+
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin:5.1
+    depends_on: ['mysql']
+    restart: unless-stopped
+    environment:
+      - PMA_HOST=mysql
+      - PMA_PORT=3306
+    ports:
+      - 8082:80
+
+  powerdns-admin:
+    image: ngoduykhanh/powerdns-admin:latest
+    depends_on: [ 'mysql' ]
+    restart: unless-stopped
+    environment:
+      - PDA_BIND_ADDRESS=0.0.0.0
+      - PDA_BIND_PORT=80
+      - PDA_DEBUG=true
+      - PDA_LOAD_ENV_FILES=true
+      - PDA_AUTOINIT_ENABLED=true
+      - PDA_AUTOINIT_FORCE=false
+      - PDA_CHECK_API_ENABLED=true
+      - PDA_CHECK_API_FAIL_DELAY=2
+      - PDA_CHECK_API_SUCCESS_DELAY=0
+      - PDA_CHECK_API_ATTEMPTS=15
+      - PDA_CHECK_PYTEST_ENABLED=false
+      - PDA_GUNICORN_ENABLED=false
+      - PDA_GUNICORN_TIMEOUT=2
+      - PDA_GUNICORN_WORKERS=2
+      - PDA_GUNICORN_LOGLEVEL=debug
+      - PDA_PDNS_API_KEY=scjfNRN4Ch2DBuhk24vr
+      - PDA_PDNS_VERSION=4.5.2
+      - PDA_PDNS_PROTO=http
+      - PDA_PDNS_HOST=pdns
+      - PDA_PDNS_PORT=8081
+      - PDA_ADMIN_USER_USERNAME=nocadmin
+      - PDA_ADMIN_USER_PASSWORD=clink henchmen mete coracle uphold
+      - PDAC_SALT=ntR8BmMQb7MKTJ5YrgQwZr3pxU3uLG
+      - PDAC_SECRET_KEY=eJ92QLuennVStHTj5FVwxz7m4r3Ng3
+      - PDAC_SIGNUP_ENABLED=true
+      - PDAC_HSTS_ENABLED=false
+      - PDAC_OFFLINE_MODE=false
+      - PDAC_FILESYSTEM_SESSIONS_ENABLED=false
+      - PDAC_SQLALCHEMY_DATABASE_URI=sqlite:////srv/app/pdns.db
+      - PDAC_SAML_ENABLED=false
+      - SQLALCHEMY_TRACK_MODIFICATIONS=true
+      - FLASK_ENV=development
+    ports:
+      - "8080:80"
+    volumes:
+      - /PATH/TO/YOUR/GIT/PROJECT/ROOT:/srv/app
+
+  pdns:
+    image: azoriansolutions/powerdns-nameserver:4.5.2-alpine-3.14-mysql
```

Comment:
What is the difference for this image with the official one ?
Using your repository is not a good practice either

Reply:
@jbe-dw I chose my own because the official image still lacks many of the common pseudo standard features that many good Docker images container. I have considered contacting their project maintainer to offer up an overhaul for their Docker images as well. This is a case where I would have used any number of third-party images over the official one for the aforementioned reasons. I just happen to have a preferred one since I created it.

Reply:
If you are interested in, you should join their irc room, they are very active. You'll have to sell the advantages at considering adding your proposed changes because they are really focused on the core development. I have tried to talk about that kind of things that may give chances to the project to get more adopters and visibility but they did not seem to care.

Reply:
@jbe-dw that is a real shame because I think we all know that when you don't stay up on the curve, everything you do follows. Again to one of your earlier points, perhaps a reflection of the project.

I will certainly put it on my list and work towards getting connected with them on the channel. Always worth one more shot for sparking the flame I suppose. Perhaps if I just deliver a completed solution ready to integrate.